src/pages/event.js

```diff
@@ -36,10 +35,9 @@ class EventPage extends React.Component {
   }
 
   async componentDidMount() {
-    const { getLoggedInUser, data } = this.props;
+    const { LoggedInUser, data } = this.props;
     const event = data.Collective;
 
-    const LoggedInUser = await getLoggedInUser();
```

Comment:
If component gets updated later (which will happen because `LoggedInUser` is fetched after the mount) then we won't get the updated `LoggedInUser`. We don't need to store the user in the state anymore, we can rely on `this.props` directly.